Vokoder.podspec

```diff
@@ -56,7 +56,9 @@ Pod::Spec.new do |s|
   s.subspec 'Swift' do |sw|
     sw.ios.deployment_target = '8.0'
     sw.tvos.deployment_target = '9.0'
-    sw.dependency 'Vokoder/DataSources'
+    sw.watchos.deployment_target = '3.0'
+    
+    sw.dependency 'Vokoder/Core'
```

Comment:
fwiw, i've also figured out that this works even if you add different subspecs to different targets.